jgrapht-ext/src/main/java/org/jgrapht/ext/DOTExporter.java

```diff
@@ -38,6 +38,8 @@
 public class DOTExporter<V, E>
     implements GraphExporter<V, E>
 {
+    private static final String DEFAULT_GRAPH_ID = "G";
```

Comment:
I think that this could be in `DOTUtils` as a public static variable, which could be reused by API users.